moshi_mlx/pyproject.toml

```diff
@@ -33,3 +33,7 @@ dev = [
     "flake8",
     "pre-commit",
 ]
+
+[project.scripts]
+moshi-server-mlx = "moshi_mlx.local_web:main"
+moshi-local = "moshi_mlx.local:main"
```

Comment:
this doesn't run, since main expects a parameter.
I could fix so that main() doesn't take a param, or make a `_main` which wraps it.

Reply:
Made a `_main` wrapper